targets/TARGET_NXP/mbed_rtx.h

```diff
@@ -97,24 +97,13 @@
 
 #elif defined(TARGET_MIMXRT1050_EVK)
 
-#if defined(__CC_ARM) || (defined(__ARMCC_VERSION) && (__ARMCC_VERSION >= 6010050))
+#if defined(__ARMCC_VERSION)
     extern uint32_t               Image$$RW_IRAM1$$ZI$$Base[];
     extern uint32_t               Image$$RW_IRAM1$$ZI$$Length[];
-    extern uint32_t               Image$$ARM_LIB_STACK$$ZI$$Base[];
-    extern uint32_t               Image$$ARM_LIB_STACK$$ZI$$Length[];
-    #define HEAP_START            ((unsigned char*) Image$$RW_IRAM1$$ZI$$Base)
-    #define HEAP_SIZE             ((uint32_t) Image$$RW_IRAM1$$ZI$$Length)
-    #define ISR_STACK_START       ((unsigned char*)Image$$ARM_LIB_STACK$$ZI$$Base)
-    #define ISR_STACK_SIZE        ((uint32_t)Image$$ARM_LIB_STACK$$ZI$$Length)
+    #define HEAP_START            Image$$RW_IRAM1$$ZI$$Base
+    #define HEAP_SIZE             Image$$RW_IRAM1$$ZI$$Length
```

Comment:
Can this be deleted as well. Looks like mbed_boot_arm_std.c has this defined.

Reply:
@mmahadevan108 - Linker script is updated to support the arm_std.c definitions. Please verify https://github.com/ARMmbed/mbed-os/pull/9571/commits/8b60339da4921f6834634995eaa654756fa5c4a6

Reply:
https://github.com/ARMmbed/mbed-os/pull/9571/commits/aa1e5561678c8be2d0a1502ad15b0c04c1443ca9 is other solution if HEAP needs to be defined (will be done in future for all targets)